src/hotspot/share/opto/macro.cpp

```diff
@@ -1837,8 +1837,8 @@ Node* PhaseMacroExpand::prefetch_allocation(Node* i_o, Node*& needgc_false,
       // This code is used to generate 1 prefetch instruction per cache line.
 
       // Generate several prefetch instructions.
-      uint step_size = AllocatePrefetchStepSize;
-      uint distance = AllocatePrefetchDistance;
+      int step_size = AllocatePrefetchStepSize;
+      int distance = AllocatePrefetchDistance;
```

Comment:
I think this is a step in the wrong direction.  Shouldn't these remain uint?

Reply:
This change also doesn't fit the PR description. There is no need to change this in this PR (or even at all).

Reply:
Okay. Reverted back.